moveit_setup_assistant/CMakeLists.txt

```diff
@@ -1,169 +1,200 @@
 cmake_minimum_required(VERSION 3.10.2)
 project(moveit_setup_assistant)
 
-if(NOT "${CMAKE_CXX_STANDARD}")
-  set(CMAKE_CXX_STANDARD 14)
-endif()
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
-set(CMAKE_CXX_EXTENSIONS OFF)
-
-if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
-  # suppress warning in Ogre
-  add_compile_options(-Wno-deprecated-register)
-endif()
+# Common cmake code applied to all moveit packages
+find_package(moveit_common REQUIRED)
+moveit_package()
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+find_package(moveit_core REQUIRED)
+find_package(moveit_ros_planning REQUIRED)
+find_package(moveit_ros_visualization REQUIRED)
+# find_package(rosconsole REQUIRED)
+find_package(rviz2 REQUIRED)
+find_package(rviz_common REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(urdf REQUIRED)
+find_package(srdfdom REQUIRED)
 
-# definition needed for boost/math/constants/constants.hpp included by Ogre to compile
-add_definitions(-DBOOST_MATH_DISABLE_FLOAT128)
-
-if(NOT CMAKE_CONFIGURATION_TYPES AND NOT CMAKE_BUILD_TYPE)
-  set(CMAKE_BUILD_TYPE Release)
-endif()
-
-find_package(Boost REQUIRED COMPONENTS thread filesystem system program_options)
-find_package(catkin REQUIRED COMPONENTS
-  moveit_core
-  moveit_ros_planning
-  moveit_ros_visualization
-  rviz
-  rosconsole
-  roscpp
-  urdf
-  srdfdom
-)
 find_package(ompl REQUIRED)
-include_directories(SYSTEM ${catkin_INCLUDE_DIRS} ${Boost_INCLUDE_DIRS} ${OMPL_INCLUDE_DIRS})
-include_directories(include)
 
 # Qt Stuff
-if(rviz_QT_VERSION VERSION_LESS "5")
-  find_package(Qt4 ${rviz_QT_VERSION} REQUIRED QtCore QtGui)
-  include(${QT_USE_FILE})
-else()
-  find_package(Qt5 ${rviz_QT_VERSION} REQUIRED Core Widgets)
-  set(QT_LIBRARIES Qt5::Widgets)
-endif()
+find_package(Qt5 ${rviz_QT_VERSION} REQUIRED Core Widgets)
+set(QT_LIBRARIES Qt5::Widgets)
+
```

Comment:
Would you mind removing any QT port from this PR and leave these sections commented out?